### ISSUE TITLE
iOS empty largeBlob

### ISSUE THREAD

**Issue by allanloji** [f-23/react-native-passkey#56]
## Issue
I'm using version 3.0.0 of the library and I want to use the largeBlob extension on iOS but on get method I'm receiving an empty object. I have registered the largeBlob with the write param on the registration options.

## Build environment
MacBook Pro, Apple M4 Pro
Expo 51.0.34 
React Native 0.74.5
"react-native-passkey": "^3.0.0",

Tested on iPad Mini iOS 18.1.1

## Bug Replicate
Create passkey with registration Options
```json
{
   "attestation":"none",
   "authenticatorSelection":{
      "requireResidentKey":true,
      "residentKey":"required",
      "userVerification":"required"
   },
   "challenge":"4VLMV8TLMnduELqagWEBwfw-1c_INufBXrEZdaHltSM",
   "excludeCredentials":[
      
   ],
   "extensions":{
      "largeBlob":{
         "supported":true,
         "write":[
            Uint8Array
         ]
      }
   },
   "pubKeyCredParams":[
      {
         "alg":-7,
         "type":"public-key"
      },
      {
         "alg":-257,
         "type":"public-key"
      }
   ],
   "rp":{
      "id":"test.app",
      "name":"Test"
   },
   "timeout":60000,
   "user":{
      "displayName":"Test",
      "id":"test@test.com",
      "name":"Test"
   }
}
```

Get Passkey
```json
{
   "allowCredentials":[
   ],
   "challenge":"4VLMV8TLMnduELqagWEBwfw-1c_INufBXrEZdaHltSM",
   "rpId":"test.app",
   "timeout":60000,
   "userVerification":"required",
   "extensions":{
      "largeBlob":{
         "read":true
      }
   }
}
```

**Comment by f-23** [comment]
@allanloji Could you post the whole result object you are seeing? That's a new one for me, but I haven't tested this on iOS 18.1+ yet so there might be some changes.

**Comment by peterferguson** [comment]
Hey @allanloji this isn't a bug, you cannot write to the largeBlob on passkey registration, only on authentication [as per the spec](https://www.w3.org/TR/webauthn-3/#sctn-large-blob-extension).

So the behaviour you are seeing is correct, you shouldn't receive confirmation that it has been written `{ written: true }` as you would in the authentication case.

Receiving `{ read: true }` in the response does look like a bug however @f-23 although I haven't tested this to see if this is indeed the result

**Comment by f-23** [comment]
@peterferguson You are totally right, I missed the part where he is trying to write to largeBlob in the registration request.

Although as far as I understand the second `Get Passkey` object @allanloji posted is meant to show his get request payload, not the result from the registration? Receiving an empty object should be the expected behaviour here.

**Comment by peterferguson** [comment]
@f-23 Oh yes sorry missed that that was a request object!

Yea think this is all good then no issue 👍
